include/dslash_helper.cuh

```diff
@@ -265,11 +265,11 @@ namespace quda
     bool remote_write;      // used by the autotuner to switch on/off remote writing vs using copy engines
 
     int_fastdiv threads; // number of threads in x-thread dimension
-    int_fastdiv exterior_threads; //  number of threads in x-thread dimension for fused exterior dslash
+    int_fastdiv exterior_threads; // number of threads in x-thread dimension for fused exterior dslash
     int threadDimMapLower[4];
     int threadDimMapUpper[4];
 
-    const bool spin_project; // whether to spin project nSpin=4 fields (generally true, except for, e.g., covariant derivative)
+    int_fastdiv Ls;
```

Comment:
Just a comment: overloading the name `Ls` might add another item the long list of terminology overloading in lattice QCD.

Reply:
🤷‍♀️